src/config.ts

```diff
@@ -6,6 +6,7 @@ const BSC_MAIN_NODE = "https://bsc-dataseed.binance.org";
 const ARB_ONE_MAIN_NODE = "https://arb1.arbitrum.io/rpc";
 const SOLANA_MAIN_NODE = "https://aged-few-shard.solana-mainnet.quiknode.pro/7823e0f3ad9d6dbefe8b0807ffc5a1119dd03a10/";
 const LENS_API_URL = "https://api.lens.dev";
+const CONFIGS_URL = `https://assets.redefined.org/configs/config-resolvers.json?v=${new Date().valueOf()}`
```

Comment:
Use `new Date().valueOf()` in request ?

Reply:
Classic avoiding of browser cache

Reply:
I know
May be need to do it in request instead of here?